README.md

```diff
@@ -1,53 +1,91 @@
 
-### Prerequisites
+## How to install the HyperShift CLI
 
-* Admin access to an OpenShift cluster (version 4.7).
-* The OpenShift `oc` CLI tool.
-* The `hypershift` CLI tool:
+The `hypershift` CLI tool helps you install and work with HyperShift.
 
-        $ make hypershift
+**Prerequisites:**
 
-### Install HyperShift
+* Go 1.16
+
+Install the `hypershift` CLI using Go:
+
+```shell
+go install github.com/openshift/hypershift@latest
+```
+
+The `hypershift` tool will be installed to `$GOBIN/hypershift`.
+
+
+## How to install HyperShift
+
+HyperShift is deployed into an existing OpenShift cluster which will host the managed control planes.
+
+**Prerequisites:**
+
+* Admin access to an OpenShift cluster (version 4.7+) specified by the `KUBECONFIG` environment variable
+* The OpenShift `oc` CLI tool
+* The `hypershift` CLI tool
 
 Install HyperShift into the management cluster:
 
-```bash
-$ bin/hypershift install
+```shell
+hypershift install
 ```
 
-Remove HyperShift from the management cluster:
+To uninstall HyperShift, run:
 
-```bash
-$ bin/hypershift install --render | oc delete -f -
+```shell
+hypershift install --render | oc delete -f -
 ```
 
-### Create an example cluster
+## How to create a hosted cluster
+
+The `hypershift` CLI tool comes with a command to help create an example hosted cluster. The cluster will come with a node pool consisting of two workers nodes.
 
-Prerequisites:
+**Prerequisites:**
 
-- A valid pull secret file for image pulls.
-- An SSH public key file for guest node access.
-- An [aws credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+- An OpenShift cluster with HyperShift installed
+- Admin access to the OpenShift cluster specified by the `KUBECONFIG` environment variable
+- The `hypershift` CLI tool
+- The OpenShift `oc` CLI tool.
+- A valid pull secret file for the `quay.io/openshift-release-dev` repository
+- An SSH public key file for guest node access
+- An [AWS credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) with permissions to create infrastructure for the cluster
 
-Install the example cluster:
+Run the `hypershift` command to generate and install the example cluster:
 
-```bash
-$ bin/hypershift create cluster \
-    --pull-secret /my/pull-secret \
-    --aws-creds /my/aws-credentials \
-    --ssh-key /my/ssh-public-key
+```shell
+hypershift create cluster \
+  --pull-secret /my/pull-secret \
+  --aws-creds /my/aws-credentials \
+  --ssh-key /my/ssh-public-key
 ```
 
-When the cluster is available, get the guest kubeconfig using:
+Eventually the cluster's kubeconfig will become available and can be fetched and decoded locally:
 
-```bash
-$ oc get secret --namespace example admin-kubeconfig --template={{.data.value}} | base64 -D
+```shell
+oc get secret \
+  --template={{.data.kubeconfig}} \
+  --namespace clusters \
+  example-admin-kubeconfig | base64 -D
```

Comment:
this is a lower `d` on the flag to `base64`

Reply:
It's -D on BSD (e.g. darwin), -d on GNU, maybe I should just omit that part as a platform-ism?

Reply:
I stand corrected, on the version of BSD `base64` that ships with 10.15.7 there is support for `-d` and `--decode` as aliases, so how about `--decode`?

Reply:
Updated to use `--decode` which hopefully is portable enough